src/tasks/shop-admin/Settings/SetSystemConfigValues.ts

```diff
@@ -1,34 +1,15 @@
 import { test as base, expect } from '@playwright/test';
 import type { Task } from '../../../types/Task';
-import type { FixtureTypes} from '../../../types/FixtureTypes';
+import type { FixtureTypes } from '../../../types/FixtureTypes';
 
 export const SetSystemConfigValues = base.extend<{ SetSystemConfigValues: Task }, FixtureTypes>({
```

Comment:
Do we need actually that task. We could directly use the TestDataService method `setSystemConfig()` in our tests.

Reply:
I've removed it